### PR TITLE
Fix crash on some valve maps

### PR DESCRIPTION
**Pull request by Kroytz**
Fix UTIL_SetModel with unprecached model

### CODE REVIEW

**Comment by Kroytz** [comment]
Tested on Latest csgo

> Protocol version 13777 [1223/1223]
> Exe version 1.37.7.7 (csgo)
> Exe build: 13:05:32 Jan  7 2021 (8012) (730)
> Windows version 6.2

File: addons/sourcemod/scripting/n_arms_fix.sp [hunk 1]
```diff
@@ -441,7 +441,7 @@ public void PrecacheModels(bool map)
     char modelsPath[64];
     Format(modelsPath, sizeof(modelsPath), "%s", g_bLegacyModels ? LEGACY_MODELS_PATH : MODELS_PATH);
 
-    if (map) {
+    if (map && HasMapModels(NTEAM_T) && HasMapModels(NTEAM_CT)) {
```
Comment (NomisCZ) [inline addons/sourcemod/scripting/n_arms_fix.sp:444]: error 017: undefined symbol "NTEAM_T" / "NTEAM_CT", you probably mean TEAM_T / TEAM_CT?

Comment (Kroytz) [reply addons/sourcemod/scripting/n_arms_fix.sp:444]: Yes. Sorry for that. I re-defined that in my local file.

**Review by NomisCZ — APPROVED** [review]

**Comment by NomisCZ** [comment]
Thank you for fixing the bug.

**Pull request merged**